feed-universe.py

```diff
@@ -322,12 +327,20 @@ def check_sync(org: Organization, datasets: list):
             active_orgs.append(org)
 
             if args.check:
-                check_sync(org, datasets)
-                continue
-
-            print(f"Feeding {len(datasets)} datasets from '{org.slug}'...")
-            for batch in batched(datasets, 1000):
-                api.put_topic_datasets(topic, batch)
+                check_sync(topic_slug, org, datasets)
+            else:
+                new_datasets += datasets
```

Comment:
Since `new_datasets` is always initialized, let's always maintain it?

Reply:
Hopefully we remember after a few more refactors ;)
Would such an init work? Makes usage more explicit.
```
new_datasets = [] if not args.check else None
```

Reply:
Nope, the typing system is not smart enough :-/

Reply:
:shrug: